test/e2e/main_test.go

```diff
@@ -34,6 +34,7 @@ var (
 	primaryNic         = flag.String("primaryNic", "eth0", "Primary network interface name")
 	firstSecondaryNic  = flag.String("firstSecondaryNic", "eth1", "First secondary network interface name")
 	secondSecondaryNic = flag.String("secondSecondaryNic", "eth2", "Second secondary network interface name")
+	reporter = NewKubernetesNMStateReporter("test_logs/e2e")
```

Comment:
seems like you did not run `go fmt` against the module

Reply:
Done